content/events/hacktoberfest.adoc

```diff
@@ -14,56 +14,52 @@ opengraph:
+is a month-long celebration of open source software.
 It happens every year in October.
-During this event everyone can support open-source by contributing changes, and then earn a limited edition swag.
-
-We invite everyone to participate and to contribute to the Jenkins project,
-regardless of their background and Jenkins experience.
-There are many ways to
-link:/participate/[contribute] to Jenkins during Hacktoberfest.
-You can work on code or documentation,
-contribute to link:https://wiki.jenkins.io/display/JENKINS/Internationalization[localization],
-create new link:/artwork[artwork], etc.
-You can also share your experiences with Jenkins by link:https://github.com/jenkins-infra/jenkins.io/blob/master/CONTRIBUTING.adoc#adding-a-blog-post[creating a new blogpost].
-Generally, any pull requests in GitHub would qualify.
-See the link:/participate/[Contribute and Participate] page for more information about how to contribute.
+During this event everyone can support open-source by contributing changes, and then earn a unique digital reward.
+This year marks the 10th anniversary of Hacktoberfest.
 
+We invite everyone to participate and to contribute to the Jenkins project, regardless of their background and Jenkins experience.
+There are many ways to link:/participate/[contribute] to Jenkins during Hacktoberfest.
+You can work on code or documentation, contribute to link:https://wiki.jenkins.io/display/JENKINS/Internationalization[localization], or create new link:/artwork[artwork], etc.
+You can also share your experiences with Jenkins by link:https://github.com/jenkins-infra/jenkins.io/blob/master/CONTRIBUTING.adoc#adding-a-blog-post[creating a new blogpost].
+Generally, any four non-spammy pull requests in GitHub or merge requests in GitLab would qualify.
+Please see the link:/participate/[Contribute and Participate] page for more information about how to contribute.
 
 == Quick start
 
 1. Sign-up to Hacktoberfest on link:https://hacktoberfest.com[the event website].
-2. Join link:https://app.gitter.im/#/room/#jenkinsci_hacktoberfest:gitter.im[our Gitter channel].
-3. Everything is set, just start creating pull-requests!
-** If a repository has no `hacktoberfest` topic set, contact the maintainers to see if they are willing to accept a contribution.
-
+2. Join link:https://matrix.to/#/#jenkinsci_hacktoberfest:gitter.im[our Gitter channel].
+3. Everything is set, just start creating pull or merge requests!
+** If a repository has no `hacktoberfest` topic is set, contact the maintainers to see if they are willing to accept a contribution.
+** Alternatively, if a pull or merge request has the label `hacktoberfest-accepted`, it will count towards your total.
+** Your pull or merge request must not be labeled as invalid to be counted.
+** For your pull or merge requests to be accepted, your pull or merge requests must be merged, or have the `hacktoberfest-accepted` label, or have an overall approving review on either GitHub or GitLab.
```

Comment:
```suggestion
** For your pull or merge requests to be accepted, your pull or merge requests must be merged, or have the `hacktoberfest-accepted` label, or have an overall approving review on GitHub.
```
The Jenkins project has no repositories on GitHub.

Reply:
You mean Gitlab, @NotMyFault ? 😉